.ci/config/matrices.yaml

```diff
@@ -13,7 +13,6 @@ engine-webapp-unit:
   - 'aws_aurora_postgresql_15'
   - 'mariadb_106'
   - 'mysql_80'
-  - 'oracle_19'
   - 'oracle_23'
```

Comment:
❌ The oracle 19 should stay for engine-webapp, check comment above `ngine-webapp-unit.databases should include all supported database versions`:
```suggestion
  - 'oracle_19'
  - 'oracle_23'
```